cache.py

```diff
@@ -8,6 +8,12 @@ def purge(*args):
     for path in args:
         run("curl -s -I -X PURGE http://localhost:7999%s | grep '200 Purged'" % path.strip())
 
+@task
+@roles('class-cache', 'class-draft_cache')
+def purge_all():
+    "Purge everything from varnish. This is done by restarting all varnish processes."
+    restart()
```

Comment:
I wonder if we should use `varnishadm` to purge, instead of restarting Varnish, e.g.:

`varnishadm ban req.url ~ .` should purge all URLs (see [here](https://opensourcehacker.com/2013/02/07/varnish-shell-singleliners-reload-config-purge-cache-and-test-hits/#Purging_Varnish_cache_from_command_line), [here](https://www.varnish-software.com/book/3/Cache_invalidation.html#naming-confusion) and [here](https://www.varnish-cache.org/lists/pipermail/varnish-dev/2012-November/007399.html) for more context).

I like the idea of staggering restarts but I'm not sure in practice how much it would help, given the long tail of content. Maybe something to test in Staging?